drivers/gpio/gpio_sim.c

```diff
@@ -0,0 +1,563 @@
+/*
+ * Copyright (c) 2020 Friedt Professional Engineering Services, Inc
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT zephyr_sim_gpio
+
+#include <device.h>
+#include <drivers/gpio.h>
+#include <drivers/gpio/gpio_sim.h>
+#include <errno.h>
+#include <stdio.h>
```

Comment:
you are not using this header(?) => do not include it

Reply:
I had used snprintf at some point. It's no longer needed.